package.json

```diff
@@ -1,20 +1,21 @@
 {
   "name": "@koale/useworker",
   "version": "2.1.2",
-  "types": "./index.d.ts",
+  "types": "dist/index.d.ts",
   "source": "src/index.js",
   "main": "dist/index.js",
   "umd:main": "dist/index.umd.js",
   "module": "dist/index.mjs",
   "scripts": {
-    "build": "microbundle src/index.js",
+    "build": "microbundle src/index.ts",
     "dev": "microbundle watch",
```

Comment:
`dev` command doesn't work @z4o4z , we need to:

- change `"source": "src/index.js"` into `"source": "src/index.ts"`

- add into `.gitignore` the generated cache directories

.gitignore:
```
.rts2_cache_*
```

Reply:
fixed
cc @alewin @IljaDaderko